contracts-core/.prettierignore

```diff
@@ -10,3 +10,4 @@ typechain/
 
 # files
 coverage.json
+*.sol
```

Comment:
*(Arbitrary comment location)*

If I go to the receive page and perform a scan with an address that has *not yet configured Umbra* I get the following error(s) after scanning completes.

<img width="929" alt="Screen Shot 2023-01-19 at 1 33 30 PM" src="https://user-images.githubusercontent.com/1479802/213534619-1f3ca5d5-eaaa-432b-bf96-946f5550d5c0.png">

Reply:
This still happens even when scanning with a private key, if the account connected has not configured Umbra.

Reply:
Should now be fixed, c774dea

Reply:
Yep! Seems to be fixed.